Modules/Authentication/Basic/BasicAuthenticationUser.cs

```diff
@@ -2,8 +2,19 @@
 
 namespace GenHTTP.Modules.Authentication.Basic;
 
-public record BasicAuthenticationUser(string Name) : IUser
+public record BasicAuthenticationUser : IUser

```

Comment:
AFAIK primary ctor can be used on records as well also in combination with params. I.e. following should work, simplifies the boilerplate code:

```C#
    public record BasicAuthenticationUser(string Name, params string[] Roles) : IUser
    {
        public string DisplayName => Name;
    }
```